docs/src/main/sphinx/release/release-389.md

```diff
@@ -0,0 +1,46 @@
+# Release 389 (6 July 2022)
+
+## General
+
+* Improve performance of queries which contain a large number of aggregations
+  that use the `row` type for representing intermediate state (e.g. `SUM`). ({issue}`12762`)
+* Improve join performance when spill to disk is not used. The old behavior for
+  the spilling join operator can be restored by setting either the
+  `force_spilling_join` configuration property or the
+  `force-spilling-join-operator` session property to false. ({issue}`12618`)
+* Remove deprecated version of `ConnectorRecordSetProvider#getRecordSet`. ({issue}`13084`)
+* Fix query failure when no arguments are passed to a table function. ({issue}`12951`)
+* Fix error where a split keeps running after query failure due to an
+  uninterruptible JONI regular expression function. ({issue}`13064`)
```

Comment:
JOIN vs JONI?

also is uninterruptible a word in meriam webster?

Reply:
JONI is correct. It's a regex library. And [yes](https://www.merriam-webster.com/dictionary/uninterruptible).

Reply:
I don't think this fixes an error. I would rephrase as:

* Fix potential resource leaks when calling long-running regular expression functions.